dev/io.openliberty.jakarta.cdi.4.0/bnd.bnd

```diff
@@ -51,15 +57,13 @@ Include-Resource: \
   @${repo;jakarta.enterprise:jakarta.enterprise.cdi-api;[4.0,5)}!/beans_3_0.xsd, \
   @${repo;jakarta.enterprise:jakarta.enterprise.cdi-api;[4.0,5)}!/beans_4_0.xsd
 
-Private-Package: com.ibm.ws.cdi.provider.service
-
-
 instrument.disabled: true
 
 publish.wlp.jar.suffix: dev/api/spec
 
 -buildpath: \
-	jakarta.inject:jakarta.inject-api;version=2.0.1,\
-	jakarta.enterprise:jakarta.enterprise.cdi-api;version=4.0.0,\
+	jakarta.inject:jakarta.inject-api;version="[2.0.1,3)",\
+	jakarta.enterprise:jakarta.enterprise.cdi-api;version="[4.0,5)",\
+	jakarta.enterprise:jakarta.enterprise.lang-model;version="[4.0,5)",\
```

Comment:
I have seen ranges used in a build path before.  Does this work?

Reply:
Yes. Also, if you just put a number, it's automatically an unbound range unless you also put `EXACT`.

Having a range is nice because it means that we just have to update oss_dependencies.maven to consume a new release.

However, for these build path entries, I think we should only allow a micro version increase. If we consume a CDI API 4.0.1 release, we'd want that in this bundle. However, if there's a CDI API 4.1, that would be a new spec version and it would go in its own bundle so we wouldn't our range here to include it.

Reply:
Yes, however we discussed in our squad and decided that the range needs to be tighter e.g. `"[4.0,4.1)"`

Reply:
As long as the 3.0 bundles do start picking up the 4.0 since those are unbounded.  We may want to improve these ranges in a lot of the files to avoid older versions picking up new ones with the unbounded version and not specifying exact.

Reply:
Just had a check through the personal build, it's definitely packaging the 3.0 jar correctly and that just lists a version number rather than a range, so I must be mistaken. We should check exactly what it does do.

Possibly I was confusing it with other bundles where we pull in the contents using `Include-Resource: @${repo;artifact:version}` where we do seem to consistently use ranges.